its/ruling/src/test/resources/expected/php-S1488.json

```diff
@@ -154,6 +154,7 @@
 789,
 841,
 960,
+1242,
```

Comment:
New issue here https://github.com/SonarCommunity/php-test-sources/blob/master/src/symfony/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php#L1242
Actually, i couldn't find out the reason for it, but it looks like true positive